tasks/publisher.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Include initial OS-specific tasks
+  ansible.builtin.include_tasks: init_{{ ansible_os_family | lower }}.yml
+  vars:
+    _cvmfs_role: publisher
+    _cvmfs_upgrade: "{{ cvmfs_upgrade_server }}"
+
+- name: Create a directory cvmfs_keys
+  ansible.builtin.file:
+    path: /tmp/keys
+    state: directory
+    mode: '0755'
+
+- name: Copy certificate key on publisher 
+  ansible.builtin.copy:
+    content: "{{ item.crt }}"
+    dest: "{{ item.path | default('/tmp/keys') }}"
```

Comment:
It is possible that `item.path` does not exist. Maybe force the path here.

Reply:
done